4337/test/eip4337/ReferenceEntryPoint.spec.ts

```diff
@@ -152,6 +155,75 @@ describe('Safe4337Module - Reference EntryPoint', () => {
     expect(await ethers.provider.getBalance(safe.address)).to.be.eq(accountBalance - transfers - deposits)
   })
 
+  it('should support a Safe signer (NOTE: would require a staked paymaster for ERC-4337)', async () => {
```

Comment:
is that because the signer would access its storage and that goes against the rules?